nbextensions/bulkImportCell/bulkImportCellStates.js

```diff
@@ -49,10 +49,8 @@ define([], () => {
         // for when the cell is in configuration mode, hasn't been run, has no jobs, no results
         editingIncomplete: {
             ui: {
-                tab: tabState(
-                    ['configure', 'info', 'jobStatus', 'results'],
-                    ['configure', 'info', 'jobStatus', 'results']
```

Comment:
`jobStatus` and `results` should not be enabled!

Reply:
I think that was in there for debugging from waaaaaay back. Time for it to go!